content/assets_c/2009/07/05-1180.md

```diff
@@ -0,0 +1,6 @@
+---
+title: 05-1180
+date: 2017-12-17
+type: asset
+---
+<img src="http://ccnmtl.columbia.edu/projects/histologylab/assets/images/05.jpg" width="800" alt="05.jpg" style="margin: 0;padding: 0;border: 0;">
```

Comment:
Just noting here in case you're not already aware: This image will display an insecure content warning under https, since it's http. I'm guessing you'll migrating these images over to this repo in the future? If so, these can all be updated at that point.